live-examples/html-examples/css/abbr.css

```diff
@@ -0,0 +1,3 @@
+abbr {
+    font-variant: all-small-caps;
```

Comment:
I wonder, should we bump the `font-size` here? Overall the text is pretty small, and specifically, it is hard to notice the dotted underline added for `abbr`

<img width="627" alt="screen shot 2018-04-12 at 10 01 06" src="https://user-images.githubusercontent.com/10350960/38663942-8fb0e9e8-3e38-11e8-8b8a-f44e79670c2e.png">

Perhaps `font-size: 1.5rem;`?

<img width="627" alt="screen shot 2018-04-12 at 10 02 29" src="https://user-images.githubusercontent.com/10350960/38663989-a59da0e8-3e38-11e8-820c-4b5842c21cef.png">

Reply:
You're right that it is too small. But isn't it weird to use small caps and then bump the font size? Perhaps it would be better to use some alternative style than small caps, like a different text color?